### ISSUE TITLE
PricingModel create returns empty result

### ISSUE THREAD

**Issue by quarryman** [gonebusy/gonebusy-nodejs-client#17]
PricingModelsController.createPricingModel() call returns next pure json response
```
{
  "pricing_model": // @notice_me - singular!
  {
    "name":"Dog",
    "notes":null,
    "id":18,
    ...
    "price":99.0,
    "currency":"USD"
  }
}
```

Response  passed to promise after being wrapped with EntitiesPricingModelResponse has empty keys

```
EntitiesPricingModelResponse: {
  currency:null,
  id:null 
  name:null
  notes:null
  ownerId:null
  price:null
  pricingModelType:null
}
```

It seems like client-library bug and presumably the reason is:

Parsed server response is passed to 'new CreatePricingModelResponse(parsed) '

[https://github.com/gonebusy/gonebusy-nodejs-client/blob/c0bd5369a610bb34dfae794bbd410b4afadf1f71/lib/Controllers/PricingModelsController.js#L218](https://github.com/gonebusy/gonebusy-nodejs-client/blob/c0bd5369a610bb34dfae794bbd410b4afadf1f71/lib/Controllers/PricingModelsController.js#L218)

CreatePricingModelResponse constructor expects data to be under plural key 'pricing_models' according to

[https://github.com/gonebusy/gonebusy-nodejs-client/blob/c0bd5369a610bb34dfae794bbd410b4afadf1f71/lib/Models/CreatePricingModelResponse.js#L19](https://github.com/gonebusy/gonebusy-nodejs-client/blob/c0bd5369a610bb34dfae794bbd410b4afadf1f71/lib/Models/CreatePricingModelResponse.js#L19)

but response has singular 'pricing_model' key instead.

**Comment by alexagranov** [comment]
Good catch @quarryman!  

The client is actually trying its best though:  the swagger.json spec is incorrect for GET /pricing_models/:id as well as POST /pricing_models/new and is using the plural as root.  :-(

I'll provide an updated version shortly.

**Comment by alexagranov** [comment]
This is now fixed in 0.0.4 (PR #18)
